src/web/tasks.py

```diff
@@ -0,0 +1,14 @@
+from celery import shared_task
+from discord import SyncWebhook
```

Comment:
Is this `discord.py`? I don't think it got added to `requirements.txt`

Reply:
Whoops, yes you are correct. Will add!

Reply:
Done: https://github.com/SpokaneTech/SpokaneTech_Py/pull/61/commits/3b5e64713120c86ef79efc4855504434e7a909d1